core/src/main/scala/kafka/raft/KafkaMetadataLog.scala

```diff
@@ -544,7 +544,7 @@ case class MetadataLogConfig(logSegmentBytes: Int,
                              retentionMillis: Long,
                              maxBatchSizeInBytes: Int,
                              maxFetchSizeInBytes: Int,
-                             fileDeleteDelayMs: Int,
+                             fileDeleteDelayMs: Long,
```

Comment:
Changed to `long` to match the type of the config (even if the default value happened to fit an int before).

Reply:
Good catch!